RUNTIMES.md

```diff
@@ -0,0 +1,22 @@
+# Supported JavaScript runtimes
```

Comment:
Do we know if this means it's actually not compatible with edge runtimes? 🤔

Reply:
I don't see any APIs here which aren't supported in every major environment type, like edge and those.